gloom/src/main.cpp

```diff
@@ -25,6 +30,8 @@ GLFWwindow* initialise()
     glfwWindowHint(GLFW_OPENGL_PROFILE, GLFW_OPENGL_CORE_PROFILE);
     glfwWindowHint(GLFW_OPENGL_FORWARD_COMPAT, GL_TRUE);
 
+    glfwSetErrorCallback(glfwErrorCallback);
```

Comment:
Move error callback below the _glfwWindowHint_ groups and write a short comment above it.